apps/mull-api/src/app/event/event.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Event } from '../entities';
+import { CreateEventInput, UpdateEventInput } from './inputs/event.input';
+@Injectable()
+export class EventService {
+  constructor(
+    @InjectRepository(Event)
+    private eventRepository: Repository<Event>
+  ) {}
+
+  findAll(): Promise<Event[]> {
+    return this.eventRepository.find();
+  }
+
+  findOne(id: number): Promise<Event> {
+    return this.eventRepository.findOne(id);
+  }
+
+  async create(eventInput: CreateEventInput): Promise<Event> {
+    const newEvent = this.eventRepository.create(eventInput);
+    await this.eventRepository.save(newEvent);
+    return newEvent;
```

Comment:
We don't need the .create() method.
```suggestion
    return await this.eventRepository.save(eventInput);
```